projects/canopy/src/lib/banner/banner.component.scss

```diff
@@ -1,13 +1,42 @@
+@import '../../styles/mixins';
+
 .lg-banner {
   display: block;
   background-color: var(--color-battleship-grey);
   color: var(--color-white);
   box-shadow: 0 0.25rem 0 0 rgba(0, 0, 0, 0.03);
+
+  a:not(.lg-btn) {
+    border-bottom-color: var(--color-white);
+    color: var(--color-white);
+
+    @include lg-link(
+      $default-color: var(--color-white),
```

Comment:
Because Canopy is fully themable, we should really use hard-coded colour variables.
In this case, what you need is to use the `light-foreground` variables that you can find listed in `styles/variables/_variants.scss`.

Before making anymore changes - have you tried applying the `lgVariant` directive to it instead? that might work 🤔 it would need a bit of cleanup in the styles and in `set variant`. Happy to jump on a call tomorrow if you want to pair?

Reply:
I did try to apply the `lgVariant` directive but the problem was the colours used in the variants aren't the same as what has been designed for the banner message component. The variant colours are more pastel than dark and bold in the banner message. Do we need to update the variant colours globally or maybe add more variants? 🤔 Yes, always happy to jump on a call and pair 🙂.

Reply:
As discussed, I have replaced the colours with the light/dark variables and created some others for the banner background and text colours.